app/models/concerns/dynamic_attribute.rb

```diff
@@ -22,7 +22,8 @@ def parse_dynamic_attribute(value)
         end
 
         # parse erb
-        parser = ERB.new(CGI.unescapeHTML(parsed_text))
+        # unescape html and decode uri-encoded charcters
```

Comment:
@Pralish this change is not supposed to be here (referenced: https://github.com/restarone/violet_rails/pull/1085)

Reply:
@donrestarone Thanks for noticing. I must have checked out from the referenced branch instead of master.